prime-router/src/main/kotlin/cli/tests/TestSubmissionsAPI.kt

```diff
@@ -0,0 +1,53 @@
+package gov.cdc.prime.router.cli.tests
+
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.extensions.authentication
+import com.github.kittinunf.fuel.json.responseJson
+import com.github.kittinunf.result.Result
+import gov.cdc.prime.router.common.Environment
+
+class TestSubmissionsAPI : CoolTest() {
+    override val name = "end2end-submission"
+    override val description = "Confirm records via submission API."
+    override val status = TestStatus.SMOKE
```

Comment:
I'm tempted to remove this e2e test for now. When the submission api is fully useful as a way to seeing what has happened to a submitted report it can be used by the other full end to end tests, which would exercise it better.